README.md

```diff
@@ -14,7 +14,11 @@ This will get you going with the latest version of dctlenv and make it easy to f
 **1. Check out dctlenv where you want**
 
 ```console
+# Install latest version
 $ git clone https://github.com/wbeuil/dctlenv ~/.dctlenv
+
+# Install a specific version (e.g. v0.1.2)
+git clone --depth 1 --branch v0.1.2 https://github.com/wbeuil/dctlenv ~/.dctlenv
```

Comment:
Can you add a `$` at the beginning of the line 21 ?

Reply:
Done